quinn-proto/src/lib.rs

```diff
@@ -274,6 +280,8 @@ pub struct Transmit {
     pub ecn: Option<EcnCodepoint>,
     /// Contents of the datagram
     pub contents: Vec<u8>,
+    /// Optional destination IP address for the datagram
+    pub dst_ip: Option<IpAddr>,
```

Comment:
Shouldn't this be `src_ip`?

Reply:
Yes. See also https://github.com/quinn-rs/quinn/pull/967#discussion_r551551134 about the naming of the helper function